AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -35,7 +35,7 @@
  */
 @interface ASStackLayoutSpec : ASLayoutSpec
 
-/** Specifies the direction children are stacked in. */
+/** Specifies the direction children are stacked in. It becomes immutable after either horizontal or vertical alignment was set */
```

Comment:
While I agree that there is no valid use case for changing the stack direction, implementing the restriction actually does not ensure the correctness of alignments, because any number of direction changes would cause problem if they are done after alignments were set. Please check [this new test case](https://github.com/facebook/AsyncDisplayKit/pull/873/files#diff-2e90ad10896b4231329ca61a0e743c63R567) if you couldn't understand the above (poorly constructed) sentence.